spec/lib/appsignal/event_formatter/active_record/sql_formatter_spec.rb

```diff
@@ -114,12 +114,6 @@
 
             it { should == ['Model load', "\"id\" = ?"] }
           end
-
```

Comment:
Why is this spec removed? Is it not a valid case? Is there a new case that should be added?